ui/helpers/constants/design-system.ts

```diff
@@ -215,13 +215,37 @@ export enum BorderStyle {
 }
 
 export enum BorderRadius {
+  /**
+   * 2px
+   */
   XS = 'xs',
+  /**
+   * 4px
+   */
   SM = 'sm',
+  /**
+   * 6px
+   */
   MD = 'md',
+  /**
+   * 8px
+   */
   LG = 'lg',
+  /**
+   * 12px
+   */
   XL = 'xl',
+  /**
+   * 0
+   */
   none = 'none',
+  /**
+   * 9999px
+   */
   pill = 'pill',
+  /**
+   * 50%
+   */
```

Comment:
Not part of this issue but very helpful to see the pixel value when hovering over the size abstraction

![Screenshot 2023-06-23 at 12 35 07 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/c8531f08-6f2a-4f41-a048-98f5b3d077da)